cluster/log-dump/README.md

```diff
@@ -0,0 +1,14 @@
+## This directory is deprecated!
+
+We have recently ported the log dumping utility from kubernetes/kubernetes repository to
```

Comment:
This will stay here for a while.
I propose:
> Log dumping utility was ported ...

Reply:
+1, recently is ambiguous in such a context

Reply:
Done.